recipe/run_conda_forge_build_setup_win

```diff
@@ -1,3 +1,7 @@
+
+rem 2 cores available on Appveyor workers: https://www.appveyor.com/docs/build-configuration/
```

Comment:
Could we please use `::`? Also let's add a line about when `CPU_COUNT` was added for pass through here too.